codegen/module_system.go

```diff
@@ -44,6 +44,7 @@ type EndpointMeta struct {
 	ClientMethodName       string
 	WorkflowPkg            string
 	ReqHeaders             map[string]*TypedHeader
+	DummyEndpoint          bool
```

Comment:
general nit: IsDummyEndpoint -- if it is a bool.  Saying DummyEndpoint gives the impression that it is the endpoint info